### DISCUSSION TITLE
How to launch a terminal process similar to `termopen()` builtin?

### DISCUSSION THREAD

**Discussion by dasupradyumna** [noib3/nvim-oxi#197] [Q&A]
There are 2 functions that neovim provides that allows launching an external process and attaching that terminal to a buffer and allowing interaction - `termopen()` and `jobstart()`. I do not see how I can get similar behavior using **nvim-oxi**.

Can anyone help me out with this?
I am really stuck with this problem.

**Comment by noib3** [comment]
Duplicate of #198.
